lib/Graph/Graph.cpp

```diff
@@ -61,11 +61,25 @@ void Module::strip() {
   }
 }
 
-void Module::clear() {
-  eraseFunctions();
+// A helper function to log the deletion of constants or placeholders of a
+// module.
+void loggingDeletionInUserFunc(const Node *n) {
```

Comment:
nit: Make this `static`

Reply:
This is intended for Storage Nodes only, so would make more sense to pass `Storage *` instead of `Node *`

Reply:
Regarding making this `static`: I think the more canonical way of doing this in C++ is to put it into an anonymous namespace, i.e.:

```
namespace {
void myFunc(...) { ... }
}
```

Reply:
Yeah, static is more C-like. We do a bit of both spread out across Glow, e.g. in `GraphOptimizer.cpp` it's all mostly static. I'm fine either way.

Reply:
put it in a namespace now